.github/workflows/beekeeper.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       REPLICA: 3
       RUN_TYPE: "PR RUN"
       SETUP_CONTRACT_IMAGE_TAG: "0.2.0"
+      BEEKEEPER_BRANCH: "auth-health-check"
```

Comment:
should we use `master` as default value?

Reply:
I actually reverted it now, this was only to test if it works.
Not bad idea to have a placeholder so that you just change it there.